README.md

```diff
@@ -1,42 +1,131 @@
-![openui5](http://openui5.org/images/OpenUI5_new_big_side.png)
+![OpenUI5 logo](http://openui5.org/images/OpenUI5_new_big_side.png)
 
 # openui5-masterdetail-app
+> [OpenUI5](https://github.com/SAP/openui5) masterdetail app using the [UI5 Build and Development Tooling](https://github.com/SAP/ui5-tooling).
```

Comment:
after this line please add the more specific text from the old readme:
This template implements a typical flexible column layout with two pages, one of the design patterns that is specified by the [SAP Fiori Design Guidelines](https://experience.sap.com/fiori-design/). It includes generic application functionality and tests that can be easily extended. You can find more documentation on this template [here](https://openui5.hana.ondemand.com/#docs/guide/8ed9339f3a99418e82a02f0fb4b5d6b9.html).